src/plone/restapi/interfaces.py

```diff
@@ -56,3 +56,36 @@ def __init__(field, context, request):
     def __call__(value):
         """Convert the provided JSON value to a field value.
         """
+
+
+class IContentListingSerializer(Interface):
+    """Adapter to serialize an IContentListing into a JSON compatible
+    representation.
+
+    This is used to produce compact content listings that only contain the
```

Comment:
Should be also perfectly usable for less compact listings, although I'm not yet sure about the reasonable use cases of having collection members in an object's representation.